### DISCUSSION TITLE
Possible bug Moralis with Web3Auth

### DISCUSSION THREAD

**Discussion by fjlogrono** [Web3Auth/Web3Auth#705] [Help]
I am using Moralis with web3Auth. In production it is working, but in localhost not. On Friday it worked, and I have not changed anything.

When I select any provider (for example a google account), in the new openlogin window that opens, the following message appears:

```
There seems to be some bug in the code. Please contact support to fix this.
Bad private key
```

I remember this happened weeks ago, but I think it fixed itself. 

Thanks

**Comment by shahbaz17** [comment]
Hey @fjlogrono 

Can you share the console and network output to better understand the issue?

**Reply by jeremymoore** [reply]
Hi @fjlogrono were you able to work this out? 

@shahbaz17 I see a similar issue. On mobile for example, we're suddenly seeing  "Error: Bad private key"
<img width="333" alt="image" src="https://user-images.githubusercontent.com/1245814/219225718-31fb27a6-48a6-4acd-95c1-15a51bd165c3.png">

Any ideas?
